smithy-typescript-codegen/src/main/java/software/amazon/smithy/typescript/codegen/TypeScriptDependency.java

```diff
@@ -74,7 +74,7 @@
 
     // Conditionally added when interacting with specific protocol test bodyMediaType values.
     AWS_SDK_QUERYSTRING_BUILDER("dependencies", "@aws-sdk/querystring-builder", "3.6.1", false),
-    XML_PARSER("dependencies", "fast-xml-parser", "3.17.4", false);
+    XML_PARSER("dependencies", "fast-xml-parser", "3.19.0", false);
```

Comment:
should this be `^3.19.0`?

Reply:
I mean, actually all of these should use either carrot or pin. I think this is outside the scope of this PR. LGTM

Reply:
Ideally, it should be caret. But we're not using caret for fast-xml-parser as they have changed LICENSE multiple times in the past and may do in future too.